src/index.js

```diff
@@ -1,20 +1,24 @@
 // inferno module
 import Inferno from 'inferno';
 
+
 // routing modules
 import { Router, Route } from 'inferno-router';
 import createBrowserHistory from 'history/createBrowserHistory';
 
 // app components
 import MyApp from './MyApp';
 import VersionComponent from './VersionComponent';
+import HeaderComponent from './HeaderComponent';
+
 
 const browserHistory = createBrowserHistory();
 
 const routes = (
 	<Router history={ browserHistory }>
 		<Route component={ MyApp }>
-			<Route path="/" component={ VersionComponent } />
+			<Route path="/" component={ VersionComponent, HeaderComponent } />
+			// <Route path="/" component={ HeaderComponent } />
```

Comment:
This is actually the routing logic, the different pages inside the app, what we want to do is create a LandingPage component and load those other components inside ( we can skip the VersionComponent, it's just an example )